creds building block

troft opened the pull request.

dtolb commented on bxml/buildingBlocks/credentials.md:
````diff
@@ -0,0 +1,18 @@
+    apiToken  : "YOUR_API_TOKEN",
+    apiSecret : "YOUR_API_SECRET"
+});
+```
````
Instead of putting the credentials **IN** the code here, replace these with the standard environment variables and add a section on how to set environment vars like [here](https://github.com/dtolb/bandwidth-call-screen#environment-variables)

```js
var client = new Bandwidth({
  userId    : process.env.BANDWIDTH_USER_ID,
  apiToken  : process.env.BANDWIDTH_API_TOKEN,
  apiSecret : process.env.BANDWIDTH_API_SECRET
};
```

this will encourage our users to follow best practices 😎

dtolb commented:
Just the one lil fix on env vars and you're good to go!

dtolb commented:
Use the triple column layout to allow for explicit code selection (JS, C#, Ruby, etc...)

troft commented:
I changed the code block and added a link to an external site on setting up environment variables. The code block is not coloring correctly and the apiToken in the description is not in line.

dtolb commented:
Preview Changes at:
        http://bw-apdocs-creds.s3-website-us-east-1.amazonaws.com/

dtolb commented:
@troft I made some changes here. I don't know why the syntax highlighting was off, but I was able to get that working 🎉 .

I added:

* Links to official documentation for various programming languages (with a small example)
* Link to how to set ENV vars for each OS
* Filler blocks for other programming languages (perhaps ask @avbel if he can provide a snip for you)

I changed
* userId as the env-var in the top block
* Links to node env vars to official documentation
* Moved `###  Credentials Code Block` to the `{% common %}` block (make *that* static for all languages)

** Please be sure to pull this branch before making changes to avoid merge conflicts 👍 **

dtolb commented:
Preview Changes at:
        http://bw-apdocs-creds.s3-website-us-east-1.amazonaws.com/

troft commented:
What else should be changed for this?

dtolb commented:
We need to fill in the code examples for C# and Ruby. Perhaps @avbel can help out with what those should look like.
